src/components/Router/MainRouter.tsx

```diff
@@ -94,7 +94,24 @@ export default class MainRouter extends Component<InternalRouterProps, State> {
       steps && steps.some(isUploadFallbackOffAndShouldUseCamera)
     const { hasCamera } = this.props
 
-    return !isDesktop && !hasCamera && shouldStrictlyUseCamera === true
+    const faceStep = steps.find(
+      (step) => step.type === 'face'
+    ) as StepConfigFace
+
+    const photoCaptureFallback =
+      faceStep?.options?.photoCaptureFallback == null
+        ? true
+        : faceStep?.options?.photoCaptureFallback
```

Comment:
You can leverage [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) here

```suggestion
    const photoCaptureFallback = faceStep?.options?.photoCaptureFallback ?? true
```